src/dataflow/src/sink/kafka.rs

```diff
@@ -129,6 +131,64 @@ impl ProducerContext for SinkProducerContext {
     }
 }
 
+#[derive(Debug)]
+pub enum SinkConsistencyState {
+    Begin,
+    End,
+}
+
+#[derive(Debug)]
+pub struct SinkConsistencyInfo {
+    topic: String,
+    schema_id: i32,
+    timestamp_counts: BTreeMap<Timestamp, i64>,
```

Comment:
any chance you could add comments to timestamp_counts and queue? It's hard to know what they refer to otherwise.